src/components/role-mapping/RoleMapping.tsx

```diff
@@ -253,36 +279,23 @@ export const RoleMapping = ({
                 }}
               />
             </ToolbarItem>
-            <ToolbarItem>
-              <Button
-                data-testid="assignRole"
-                onClick={() => setShowAssign(true)}
-              >
-                {t("common:assignRole")}
-              </Button>
-            </ToolbarItem>
-            <ToolbarItem>
-              <Button
-                variant="link"
-                data-testid="unAssignRole"
-                onClick={toggleDeleteDialog}
-                isDisabled={selected.length === 0}
-              >
-                {t("common:unAssignRole")}
-              </Button>
-            </ToolbarItem>
+            <ManagerToolbarItems />
           </>
         }
-        actions={[
-          {
-            title: t("common:unAssignRole"),
-            onRowClick: async (role) => {
-              setSelected([role]);
-              toggleDeleteDialog();
-              return false;
-            },
-          },
-        ]}
+        actions={
+          isManager
+            ? [
+                {
+                  title: t("common:unAssignRole"),
+                  onRowClick: async (role) => {
```

Comment:
There is nothing asynchronous happening here, so I believe you can omit the `async` keyword.
```suggestion
                  onRowClick: (role) => {
```

Reply:
That was not my code, but I think whoever wrote it was correct.  When I remove the `async` I get `Type 'boolean' is not assignable to type 'void | Promise<boolean>`

Reply:
Ah, yes. That's fine then.